src/diffpy/utils/__init__.py

```diff
@@ -20,5 +20,3 @@
 
 # silence the pyflakes syntax checker
 assert __version__ or True
-
-# End of file
```

Comment:
Remove `# End of file` as no longer needed. Unless I am not sure it has a specific usecase.